pkg/app/command/runner.go

```diff
@@ -0,0 +1,111 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/srvc/ery/pkg/domain"
+	"github.com/srvc/ery/pkg/util/netutil"
+	"go.uber.org/zap"
+)
+
+type Runner interface {
+	Run(ctx context.Context, name string, args []string) error
+}
+
+func NewRunner(
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewRunner should have comment or be unexported